beta-src/src/components/map/components/WDTerritory.tsx

```diff
@@ -82,6 +82,9 @@ const WDTerritory: React.FC<WDTerritoryProps> = function ({
       setTerritoryStrokeOpacity(1);
       deleteCommand(key);
     },
+    DISBAND: (command) => {
+      console.log(command);
+    },
```

Comment:
Why are you thinking this should do? Not sure why it's here.

Reply:
I was following the trace of events when disbanding. I am thinking there may be something that needs to happen here since clicking on unit and then territory disbands. Perhaps removing the unit from the country in state